tsconfig.json

```diff
@@ -2,24 +2,62 @@
   "compilerOptions": {
     "target": "ES2020",
     "useDefineForClassFields": true,
-    "lib": ["ES2020", "DOM", "DOM.Iterable"],
+    "lib": [
+      "ES2020",
+      "DOM",
+      "DOM.Iterable"
+    ],
     "module": "ESNext",
     "skipLibCheck": true,
-
     /* Bundler mode */
     "moduleResolution": "bundler",
     "allowImportingTsExtensions": true,
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-
     /* Linting */
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "baseUrl": ".",
+    "paths": {
+      "@todo/*": [
+        "./src/*",
+        "./dist/*",
+        ""
+      ],
+      "@todo/api": [
+        "src/api/*"
+      ],
+      "@todo/utils/*": [
+        "src/utils/*"
+      ],
+      "@todo/store/*": [
+        "src/store/*"
+      ],
+      "@todo/setup/*": [
+        "src/setup/*"
+      ],
+      "@todo/components/*": [
+        "src/components/*"
+      ],
+      "@todo/hooks/*": [
+        "src/hooks/*"
+      ],
+      "@todo/consts": [
+        "src/consts.ts"
+      ],
+    }
   },
-  "include": ["src"],
-  "references": [{ "path": "./tsconfig.node.json" }]
-}
+  "include": [
+    "src",
+    "types"
+  ],
+  "references": [
+    {
+      "path": "./tsconfig.node.json"
+    }
+  ]
+}
```

Comment:
eol

Reply:
Prettier can not allow to add eol here, such situation with eslitrc.